### PR TITLE
Hide stderr of mpv

### LINKED ISSUE #361: [BUG] mov-cli shows MPV logs, making Replay/Quit controls offset and break after input
**Describe the bug:**
During and after watching a video (with MPV), the Quit/Replay controls are offset one line up, breaking after pressing any arrow key.

This offset is increased when logs from MPV are outputted.

**Mov-cli version:** ``4.4.14``

**Expected behaviour:**

The controls should not be offset and MPV logs should not be shown unless an error occurs.

**Steps to reproduce:**

1. Watch a YouTube video with mov-cli
2. Observe the issue.

**Screenshots:**

Here's an Asciinema video of the bug: (Actual bug is shown at 00:16)
[asciicast](https://asciinema.org/a/677928)

Here's a screenshot:
Screenshot

**Logs:**

```python
[DEBUG] (mov_cli) - Config -> {'version': 1, 'debug': True, 'player': 'mpv', 'quality': 'auto', 'skip_update_checker': False, 'auto_try_next_scraper': True, 'hide_ip': True, 'ui': {'preview': True, 'watch_options': True, 'display_quality': False}, 'plugins': {'test': 'mov-cli-test', 'youtube': 'mov-cli-youtube'}, 'scrapers': {'test': 'test.DEFAULT', 'default': 'youtube'}, 'http': {'timeout': 15}}

Good Afternoon, yesseruser.
    It's 04:56 PM on a beautiful Friday!

[DEBUG] (mov_cli) - Scraper args picked up on --> []
[INFO] (mov_cli) - Using 'youtube.DEFAULT' scraper...
[INFO] (mov_cli) - Searching for 'steven he'...
[youtube:search] Extracting URL: ytsearch70:steven he
[download] Downloading playlist: steven he
[youtube:search] query "steven he": Downloading web client config
[youtube:search] query "steven he" page 1: Downloading API JSON
[youtube:search] query "steven he" page 2: Downloading API JSON
[youtube:search] query "steven he" page 3: Downloading API JSON
[youtube:search] query "steven he" page 4: Downloading API JSON
[youtube:search] Playlist steven he: Downloading 70 items of 70
[download] Downloading item 1 of 70
[download] Downloading item 2 of 70
[download] Downloading item 3 of 70
[download] Downloading item 4 of 70
[download] Downloading item 5 of 70
[download] Downloading item 6 of 70
[download] Downloading item 7 of 70
[download] Downloading item 8 of 70
[download] Downloading item 9 of 70
[download] Downloading item 10 of 70
[download] Downloading item 11 of 70
[download] Downloading item 12 of 70
[download] Downloading item 13 of 70
[download] Downloading item 14 of 70
[download] Downloading item 15 of 70
[download] Downloading item 16 of 70
[download] Downloading item 17 of 70
[download] Downloading item 18 of 70
[download] Downloading item 19 of 70
[download] Downloading item 20 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 22 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 24 of 70
[download] Downloading item 25 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 27 of 70
[download] Downloading item 28 of 70
[download] Downloading item 29 of 70
[download] Downloading item 30 of 70
[download] Downloading item 31 of 70
[download] Downloading item 32 of 70
[download] Downloading item 33 of 70
[download] Downloading item 34 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 36 of 70
[download] Downloading item 37 of 70
[download] Downloading item 38 of 70
[download] Downloading item 39 of 70
[download] Downloading item 40 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 42 of 70
[download] Downloading item 43 of 70
[download] Downloading item 44 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 46 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 49 of 70
[download] Downloading item 50 of 70
[download] Downloading item 51 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 53 of 70
[download] Downloading item 54 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 56 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 65 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Downloading item 69 of 70
[download] Video is a YouTube short. Pass '--shorts' to the scraper to scrape for them.
[download] Finished downloading playlist: steven he
[DEBUG] (mov_cli) - [Cache] Clearing all cache in file or section...
[INFO] (mov_cli) - Scraping 'Youtube and Mrbeast Right Now ~ Steven He'...
[youtube] Extracting URL: https://www.youtube.com/watch?v=Oq3xjkEOPpg
[youtube] Oq3xjkEOPpg: Downloading webpage
[youtube] Oq3xjkEOPpg: Downloading ios player API JSON
[youtube] Oq3xjkEOPpg: Downloading web creator player API JSON
[youtube] Oq3xjkEOPpg: Downloading m3u8 information
[DEBUG] (mov_cli) - [Cache] Setting 'https://www.youtube.com/watch?v=Oq3xjkEOPpg' cache...
[INFO] (mov_cli) - Playing 'Youtube and Mrbeast Right Now ~ Steven He' with MPV...
[DEBUG] (mov_cli) - Called player with these args -> 'mpv https://rr2---sn-2gb7sn7y.googlevideo.com/videoplayback?expire=1727470604&ei=rMf2ZorQFti06dsPoPGf-As&ip={the-cat-snatched-your-ip-address}&id=o-AIzlXCmsYtcLhLRCJkyz9Su8OZ6phW5ZFOD8Kus_cbT-&itag=401&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=xB&mm=31%2C26&mn=sn-2gb7sn7y%2Csn-c0q7lnz7&ms=au%2Conr&mv=m&mvi=2&pl=17&initcwndbps=1607500&vprv=1&svpuc=1&mime=video%2Fmp4&rqh=1&gir=yes&clen=141951037&dur=212.795&lmt=1727447383737428&mt=1727448547&fvip=3&keepalive=yes&fexp=51299152%2C51300761&c=IOS&txp=5532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cvprv%2Csvpuc%2Cmime%2Crqh%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRAIgb5laR3TSZ5pzfBXsFEyC4odKOeOAK2ag0JoxJwHjvDACIGZcOrE5ETQ0eN6_9TAfHcIsJ8YQP9iRrHYhA3OfVJfi&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=ABPmVW0wRgIhAPyNGvO7oa13bTq-KnDMqUiGmxYrQk1I_893SpLkJ7o1AiEAwdueR-ZZXOq9Cv1RpJcvEw-pOJWTeGztuktkUvdrD6k%3D --audio-file=https://rr2---sn-2gb7sn7y.googlevideo.com/videoplayback?expire=1727470604&ei=rMf2ZorQFti06dsPoPGf-As&ip={the-cat-snatched-your-ip-address}&id=o-AIzlXCmsYtcLhLRCJkyz9Su8OZ6phW5ZFOD8Kus_cbT-&itag=140&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=xB&mm=31%2C26&mn=sn-2gb7sn7y%2Csn-c0q7lnz7&ms=au%2Conr&mv=m&mvi=2&pl=17&initcwndbps=1607500&vprv=1&svpuc=1&mime=audio%2Fmp4&rqh=1&gir=yes&clen=3445899&dur=212.857&lmt=1727440954206617&mt=1727448547&fvip=3&keepalive=yes&fexp=51299152%2C51300761&c=IOS&txp=5532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cvprv%2Csvpuc%2Cmime%2Crqh%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRQIhAKxO0f7FuJKFIYWfiZQXnApndKbGqGHaBwX7dgGsYCj2AiAQc45WvXsioKCpCm9WxbCRr3IdevTcCzpGvqSv9DpgHA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=ABPmVW0wRgIhAPyNGvO7oa13bTq-KnDMqUiGmxYrQk1I_893SpLkJ7o1AiEAwdueR-ZZXOq9Cv1RpJcvEw-pOJWTeGztuktkUvdrD6k%3D --force-media-title=Youtube and Mrbeast Right Now ~ Steven He --sub-file=https://www.youtube.com/api/timedtext?v=Oq3xjkEOPpg&ei=rMf2Zt2DHoSJ6dsPoKPgsAY&caps=asr&opi=112496729&exp=xbt&xoaf=4&hl=en&ip={the-cat-snatched-your-ip-address}&ipbits=0&expire=1727474204&sparams=ip%2Cipbits%2Cexpire%2Cv%2Cei%2Ccaps%2Copi%2Cexp%2Cxoaf&signature=AC7BB6BEBABCB3A8AC991B4C60E79FE90D488CE6.40AD4A8BAF53345035B5F14E7DE42DA2557CB6A0&key=yt8&kind=asr&lang=en&fmt=vtt --no-terminal'
```

**Platform:**

 - OS: Ubuntu
 - Version: 24.04.1 LTS

**Additional context / possible cause or solution:**

This bug appears on Warp terminal and GNOME Terminal.

It also appears on Mac with Warp and iTerm.

**Checklist**
- [X] I have added logs.
- [X] I have added my platform.
- [X] I have added my version number.
- [X] I am sure that this issue is not related to a plugin.

### PR DESCRIPTION
**Pull request by yesseruser**
Fixes #361.

### CODE REVIEW

File: mov_cli/players/mpv.py
Comment (THEGOLDENPRO) [inline mov_cli/players/mpv.py]: We only want the stdout and stderr to be eaten when mov-cli is not in debug mode, similar to what we did to [here](https://github.com/mov-cli/mov-cli/pull/362/commits/c05b64bb560eafdf5ca0295be6b57b99f608a2ae#diff-026d99f365876dcf4e513f759692d1406291cde3409e726daa12c4570298fbe0R55-R56).

Also are you sure this works? I would have:
```python
stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT
```

**Review by THEGOLDENPRO — CHANGES_REQUESTED** [review]
Can you check out my comment? Once you complete that I'm happy to merge. 😊

File: mov_cli/players/mpv.py
Comment (yesseruser) [reply mov_cli/players/mpv.py]: I suppose your version is better. But not eating stdout in debug mode would mean the bug would still be present.

PIPE should eat the output, but return it as a tuple (stdout, stderr) from the Popen function. Is it being printed after returning and exiting?

**Comment by THEGOLDENPRO** [comment]
Thanks, I'll confirm and merge when I get home later today. 👍

File: mov_cli/players/mpv.py
Comment (THEGOLDENPRO) [reply mov_cli/players/mpv.py]: Oh sorry, I didn't see your comment earlier when I was on github mobile. Anyways about your comment, I had forgotten what PIPE did that's why I was unsure on your implementation earlier; thanks for informing me.

>  Is it being printed after returning and exiting?

Oh yeah also no we are not using the output from the tuple anywhere in mov-cli at the moment so `DEVNULL` also works.

**Comment by THEGOLDENPRO** [comment]
Just tested the pr and everything works as expected, so I'm now merging this. I'm also gonna back port this fix to v4.4 so you don't have to wait for the next major release.

**Pull request merged**
